### PR TITLE
Remove legacy tests from Jenkins, build python 3 deb packages

### PR DESCRIPTION
**Pull request by vaporos**

### CODE REVIEW

**Comment by vaporos** [comment]
This PR does not remove a lot of legacy code, that will come in a future PR.  This one is more about shifting to building/testing only python3 packages and updating Jenkins accordingly.

File: Jenkinsfile [hunk 1]
```diff
@@ -57,49 +57,12 @@ node ('master') {
...
 
         // Run the tests
-        stage("Run 0-8 Tests"){
+        stage("Run Tests") {
             // Required docker containers are built by the tests
             sh './bin/docker_build_all -p $(printf $BUILD_TAG | sha256sum | cut -c1-64)'
             sh './bin/run_tests -p $(printf $BUILD_TAG | sha256sum | cut -c1-64)'
         }
 
-        stage("Remove Docker Images") {
```
Comment (peterschwarz) [inline Jenkinsfile]: Should this have been removed as well?

File: sdk/python/setup.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Intel Corporation
```
Comment (peterschwarz) [inline sdk/python/setup.py]: License year

File: signing/setup.py [hunk 3]
```diff
@@ -72,7 +72,9 @@ def version(default):
     author='Mic Bowman, Intel Labs',
```
Comment (peterschwarz) [inline signing/setup.py]: Is this still an accurate author list?

**Comment by vaporos** [comment]
protobuf reorg is in this PR as it is required to build the deb files.

File: sdk/python/setup.py (hunk 2 above)
Comment (vaporos) [inline sdk/python/setup.py]: Updated to "2016, 2017" (this file is derived from other files.

File: signing/setup.py (hunk 3 above)
Comment (vaporos) [inline signing/setup.py]: Will update this, descriptions, versioning in another PR.  This wasn't required to disable legacy tests or to build python 3 deb files, so I did not update it here.

File: Jenkinsfile (hunk 1 above)
Comment (vaporos) [inline Jenkinsfile]: I think so, as a similar stage "Remove docker images" removes both the build and docs images.

**Comment by vaporos** [comment]
@aludvik - should we do that in a different move-docker-images PR, or is this related to changes here?

**Comment by aludvik** [comment]
We can do it in another PR

**Pull request merged**